db/version_set.cc

```diff
@@ -3386,11 +3386,22 @@ Status VersionSet::ApplyOneVersionEdit(
           edit.column_family_name_);
     }
     auto cf_options = name_to_options.find(edit.column_family_name_);
-    if (cf_options == name_to_options.end()) {
+    // implicitly add persistent_stats column family without requiring user
+    // to specify
+    bool is_persistent_stats_column_family =
+        edit.column_family_name_.compare(kPersistentStatsColumnFamilyName) == 0;
+    if (cf_options == name_to_options.end() &&
+        !is_persistent_stats_column_family) {
       column_families_not_found.insert(
           {edit.column_family_, edit.column_family_name_});
     } else {
-      cfd = CreateColumnFamily(cf_options->second, &edit);
+      // recover persistent_stats CF from a DB that already contains it
+      // TODO(Zhongyi): find out a way to pass cf options
+      if (is_persistent_stats_column_family) {
+        cfd = CreateColumnFamily(ColumnFamilyOptions(), &edit);
```

Comment:
Are we going to use default values for the persistent stats column family? I feel many option values for persistent stats column family can differ from the default because the column family itself is kind of special.